tests/test_config_parsing/config_dict_generator.py

```diff
@@ -312,7 +332,13 @@ def to_config_file(filename, config_dict):
                 for setting in keyword_value:
                     config.write(
                         f"{keyword}"
-                        f" {setting[ConfigKeys.NAME]} {setting[ConfigKeys.VALUE]}\n"
```

Comment:
and this was missing the queue system name, only writing the option name and value, right? i'm just asking to see if i understand correctly

Reply:
... and that's why we were getting all these DRIVER:... not recognized logs!

Reply:
Yes :)

Reply:
cool ^^